docs/apache-airflow/extra-packages-ref.rst

```diff
@@ -343,27 +350,278 @@ you need to list explicitly all the non-bundle extras that you want to install.
 +---------------------+-----------------------------------------------------+------------------------------------------------------------------------+
 | all_dbs             | ``pip install 'apache-airflow[all_dbs]'``           | All database integrations                                              |
 +---------------------+-----------------------------------------------------+------------------------------------------------------------------------+
-| devel               | ``pip install 'apache-airflow[devel]'``             | Minimum development dependencies (without Hadoop, Kerberos, providers) |
-+---------------------+-----------------------------------------------------+------------------------------------------------------------------------+
-| devel_hadoop        | ``pip install 'apache-airflow[devel_hadoop]'``      | Adds Hadoop stack libraries to ``devel`` dependencies                  |
+
+Development extras
+------------------
+
+Generally none of the ``devel`` extras install providers - they expect the providers to be used from sources
```

Comment:
Propose to clearly separate it from the overall `devel` name.
```suggestion
Generally none of the ``devel_*`` extras install providers - they expect the providers to be used from sources
```

Reply:
It's gone now. We have now very, very few `devel_` extras after removing the `devel_` and `editable_` provider extras.